posix.mak

```diff
@@ -234,15 +237,13 @@ $(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.dvi
 # Git clone rules
 ################################################################################
 
-# LATEST
-../%-${LATEST}/.cloned :
-	[ -d $(@D) ] || git clone ${GIT_HOME}/$* $(@D)/
-	if [ -d $(@D)/.git ]; then cd $(@D) && git checkout v${LATEST}; fi
-	touch $@
-
-# HEAD
+# dmd, druntime, dmd-2.066.1, dub-0.9.21
 ../%/.cloned :
-	[ -d $(@D) ] || git clone ${GIT_HOME}/$* $(@D)/
+ifneq ('',$(word 2,$(subst -, ,$*))) # clone a specific version
```

Comment:
Are you sure this isn't `ifneq (,$(word 2,$(subst -, ,$*)))`

In other words, where do those quotes come about?